pyaerocom/dynamic_rec_array.py

```diff
@@ -75,10 +75,8 @@ def data(self) -> np.array:
 
         :return: np.aarry of type dtype
         """
-        if self._capacity != self._length:
-            self._array = self._array[:][: self._length]
-            self._capacity = len(self._array)
-        return self._array
+        # NOTE: Commenting out the shrinking of the array might be needed in debug mode
+        return self._array[:][: self._length]
```

Comment:
You note that "Commenting out the shrinking of the array might be needed in debug mode", but it seems that you have completely removed this feature. Is this on purpose?

Reply:
Removed